langchain_opendataloader_pdf/document_loaders.py

```
@@ -245,48 +255,60 @@ def lazy_load(self) -> Iterator[Document]:
+                opendataloader_pdf.convert(
+                    input_path=self.file_paths,
+                    output_dir=output_dir,
+                    **convert_kwargs,
+                    markdown_page_separator=page_sep,
+                    text_page_separator=page_sep,
+                    html_page_separator=page_sep,
+                )
+            except Exception as e:
+                if self.hybrid:
+                    raise
+                logger.error(f"Error during conversion: {e}")
```

Ruff (0.15.9)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)